ideascube/conf/kb_fra_crfaix.py

```diff
@@ -0,0 +1,13 @@
+"""KoomBook conf"""
+from .kb import *  # pragma: no flakes
+
+LANGUAGE_CODE = 'fr'
+IDEASCUBE_NAME = 'CRF AIX'
```

Comment:
> They are not really necessary actually, but because I can specify variable I did it...

I'd rather you didn't, then.

Eventually we want to move to a point where we have almost no site-specific configuration file in Ideascube, and the first step is to stop adding new ones unless necessary.